### ISSUE TITLE
Root .eslintrc.json not generated for workspaces without a default app

### ISSUE THREAD

**Issue by christianliebel** [angular-eslint/angular-eslint#936]
Adding angular-eslint via `ng add` does not work out of the box if the workspace has no default application (i.e., when it was created with `--create-application=false`). It looks like the _.eslintrc.json_ in the root directory is not being generated.

## Steps to reproduce
```
ng new temp --create-application=false
cd temp
ng add app temp
ng add @angular-eslint/schematics
ng lint
```

## Current behavior
The following error message appears:

```
An unhandled exception occurred: Failed to load config "../../.eslintrc.json" to extend from.
Referenced from: /Users/…/temp/projects/temp/.eslintrc.json
```

## Expected behavior

Linting should work normally.

**Comment by rdebeasi** [comment]
Possibly related: #462

**Comment by JamesHenry** [comment]
Thanks folks, this is resolved in `13.2.0`
